xc/device_common.mli

```diff
@@ -74,6 +74,10 @@ val list_backends : xs:Xenstore.Xs.xsh -> Xenctrl.domid -> device list
     directory.*)
 val list_frontends : xs:Xenstore.Xs.xsh -> ?for_devids:int list -> Xenctrl.domid -> device list
 
+val make_nsv_device : ?back_domid:int -> int -> int -> device
+
+val list_xenserver_devices: xs:Xenstore.Xs.xsh -> ?for_devids:int list -> Xenctrl.domid -> device list
```

Comment:
Please add docstrings explaining what these functions are doing and the meaning of the parameters